.eslintrc.json

```diff
@@ -20,7 +20,8 @@
     ],
     "@tanstack/query/exhaustive-deps": "error",
     "@tanstack/query/no-rest-destructuring": "error",
-    "@tanstack/query/stable-query-client": "error"
+    "@tanstack/query/stable-query-client": "error",
+    "prettier/prettier": "error"
```

Comment:
why do we need this?

Reply:
will ensure prettier rules are followed

which includes tailwindcss
tailwindcss classNames have a heriarchy and formating a file with prettier will sort the classNames as well.

good for readibility.

Reply:
but it already does that?

Reply:
i am reverting this change; i can see the classname heirarchy being followed without prettier being added here